doc/src/main/asciidoc/inc/_enricher.adoc

```diff
@@ -959,6 +963,10 @@ The following configuration parameters can be used to override the behavior of t
 | The `path` part of the URL where the service is hosted.
 | `/`
 
+| *path*
```

Comment:
"port" ?

Also, I would explain the lookup mechanism here, e.g. that a service is inspected